CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+## 0.99.1
+
+ISSUES FIXED:
+
+* [48](https://github.com/perfectsense/gyro-azure-provider/issues/48): Refactor Sql (QA, ResourceFinder, Direct Dependency, Copyable).
```

Comment:
This should be removed since that PR is not yet merged. Might be tricky because this commit was cherry-picked. I guess we'll just have to accept the merge conflict later.